tests/conftest.py

```diff
@@ -20,3 +24,10 @@ def pytest_configure() -> None:
     database_name = cluster["database"]
     connection.execute(f"DROP DATABASE IF EXISTS {database_name};")
     connection.execute(f"CREATE DATABASE {database_name};")
+
+
+@pytest.fixture(params=[0, 100], ids=["legacy", "ast"])
+def ast(request) -> None:
```

Comment:
This name kind of implies that the test is only being run with the AST query, not both types. There's probably something more descriptive that could be used here

Reply:
ok